### ISSUE TITLE
Max Batch Size Estimation

### ISSUE THREAD

**Issue by wewe0901** [myelintek/web#44]
When user Choose Faster training configuration, We should choose max batch size for user.
- Run Training on single gpu and try to test the boundary for Max Batch size
- According Max Batch Size to Adjust Learning Rate 
    - ![img_4477](https://user-images.githubusercontent.com/2045106/44011075-fd37690e-9ee8-11e8-95d0-47f068990517.JPG)
    - (refer to facebook's paper)
